### PR TITLE
Modification in Address provider and added Person provider in en_PH

### PR DESCRIPTION
**Pull request by arielmagbanua**
Modification in Address provider and added Person provider in en_PH

### CODE REVIEW

**Comment by arielmagbanua** [comment]
Thanks, will be adding newer barangays and organize the arrays in the Person provider.

File: src/Faker/Provider/en_PH/Person.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
...', 'Stephen', 'Stephon', 'Sterling', 'Steve', 'Stevie', 'Stewart', 'Stone', 'Stuart', 'Sydney', 'Sylvan', 'Sylvester', 'Tate', 'Taylor', 'Ted', 'Terence', 'Terrence', 'Terry', 'Thaddeus', 'Theo', 'Theodore', 'Theron', 'Thomas', 'Thurman', 'Tillman', 'Timmothy', 'Timmy', 'Timothy', 'Tito', 'Titus', 'Tobin', 'Toby', 'Tod', 'Tom', 'Tomas', 'Tommie', 'Toney', 'Toni', 'Tony', 'Torey', 'Torrance', 'Torrey', 'Toy', 'Trace', 'Tracey', 'Travis', 'Travon', 'Trent', 'Trey', 'Triston', 'Troy', 'Trystan', 'Turner','Tyler', 'Tyree', 'Tyreek', 'Tyrel', 'Tyrell', 'Tyrese', 'Tyrique', 'Tyshawn', 'Tyson','Ubaldo', 'Ulices', 'Ulises', 'Unique', 'Uriah', 'Valentin', 'Van', 'Vance', 'Vaughn', 'Vern', 'Vicente', 'Victor', 'Vidal', 'Vince', 'Vincent', 'Vincenzo', 'Vinnie', 'Virgil', 'Vladimir', 'Wade', 'Waldo', 'Walker', 'Wallace', 'Walter', 'Walton', 'Ward', 'Warren', 'Watson', 'Waylon', 'Wayne', 'Webster', 'Weldon', 'Wendell', 'Westley', 'Weston', 'Wilber', 'Wilbert', 'Wilburn', 'Wiley', 'Wilford', 'Wilfred', 'Wilfredo', 'Wilfrid', 'Will', 'Willard', 'William', 'Willis', 'Willy', 'Wilmer', 'Wilson', 'Wilton', 'Winston', 'Xavier', 'Xander','Zachariah', 'Zachary', 'Zachery', 'Zack', 'Zander', 'Zechariah'
```
Comment (fzaninotto) [inline src/Faker/Provider/en_PH/Person.php]: please add a reference explaining where you took the data from, and split the line by first letter for easier readability. Same for other name arrays in this class.

**Comment by fzaninotto** [comment]
Bump, I will close this PR shortly without any feedback on my comment.

**Comment by arielmagbanua** [comment]
Hi... sorry for the late response I'm quite busy lately... But I will update my work soon. You can close the PR though I will just push my changes ang make a new PR. Thanks.

**Comment by fzaninotto** [comment]
I'd prefer that you push changes to this branch. That would ease my review process.

**Pull request closed without merging**